util-taglib/src/com/liferay/taglib/aui/base/BaseInputTag.java

```diff
@@ -573,6 +595,8 @@ protected void setAttributes(HttpServletRequest request) {
 	private static final String _PAGE =
 		"/html/taglib/aui/input/page.jsp";
 
+	private List<String> _activeLocales = null;
```

Comment:
I would try to initialize it to ArrayList. I've seen that several tags do not initialize (or do it in the constructor) and others do (https://github.com/liferay/liferay-portal/blob/991e5c1030b6382d07fb1bf4a5ef9fdb3ebc329c/modules/apps/item-selector/item-selector-taglib/src/main/java/com/liferay/item/selector/taglib/servlet/taglib/RepositoryEntryBrowserTag.java#L373), it's cleaner to init to ArrayList. I don't know if Brian would like it but it's better.

Reply:
Ok, I have to change a little bit the logic in the Tag jsp since we're checking if `activeLocales` is not null instead of is empty, but shouldn't be hard.

Reply:
Done! https://github.com/liferay-frontend/liferay-portal/pull/1328/commits/1d121eba6d16c6a479e3cc0509b9bbd839483e99